HopsanGUI/HcomHandler.cpp

```diff
@@ -4718,6 +4726,40 @@ void HcomHandler::executeConnectCommand(const QString cmd)
 }
 
 
+//! @brief Execute function for "unco" command
+void HcomHandler::executeListUnconnectedCommand(const QString cmd)
+{
+    QStringList args = splitCommandArguments(cmd);
+    QString wildcard = "*";
+    if(args.size() == 1)
+    {
+        wildcard = args[0];
+    }
+    else if(args.size() > 1)
+    {
```

Comment:
It would be helpful if this text mentioned how many are supported, or maybe list the description.